api/notd/manager.py

```diff
@@ -281,7 +282,13 @@ async def retreive_token_metadata(self, registryAddress: str, tokenId: str) -> R
             ], limit=1,
         )
         if len(tokenMetadatas) == 0:
-            raise NotFoundException()
+            tokenMetadatas = []
```

Comment:
i dont think you need this

Reply:
But how do we keep  the return statement the same  the tokenMetadatas[0]

Reply:
isnt `tokenMetadatas` already an empty list at this point?

Reply:
Ohh